### PR TITLE
Replace default App with routes to Film and 404

### PR DESCRIPTION
**Pull request by nilssonemil**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,25 +1,19 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch
+} from 'react-router-dom'
+import { Film, PageNotFound } from './routes'
```
Comment (Godnoken) [inline src/App.tsx]: Interesting... Just looking at the files here on github, I'm not too sure how this works.
Importing Film and PNF from "./routes"?

Comment (Godnoken) [reply src/App.tsx]: Noticed it's a folder now, oooops

Comment (nilssonemil) [reply src/App.tsx]: `./routes` contain an index.ts file which exports all exports of both `./routes/Film` and `./routes/PageNotFound`, and as `./routes` and `./routes/index` are "synonyms" we can use imports without knowing exactly which file they are from.

File: src/routes/index.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+export * from './Film'
+export * from './PageNotFound'
```
Comment (Godnoken) [inline src/routes/index.ts]: Hmmmm, is it standard practice to have an index file for each folder of components?

**Review by Godnoken — APPROVED** [review]

File: src/routes/index.ts (hunk 2 above)
Comment (nilssonemil) [reply src/routes/index.ts]: No clue. But I like it, as it can expose other components so you do not have to specify exactly where they are from.

**Pull request merged**